src/search.ts

```diff
@@ -99,8 +100,53 @@ function getRequiredModules(requireNodes: Node[]): SearchValue {
       // If expression is not a literal or a template literal, it is dynamically
       // evaluated
     } else {
-      dynamicEvalPos.push(pos);
+      dynamicArgPos.push(pos);
+    }
+  });
+  return {requiredModules, dynamicArgs: dynamicArgPos, dynamicRequire: []};
```

Comment:
`dynamicRequire` isn't filled with any kind of useful information in this function. Is it necessary to return an empty array here? What alternatives might there be?

Reply:
The ```getRequiredModules``` can return an ```x``` that contains only the ```requiredModules``` and ```dynamicArgs``` properties. And in ```search```, we can use ```Object.assign(a, b)``` where a is an empty searchObject and b is x.

Reply:
or ```getRequiredModules``` can return an array containing the two properties..